data/card_catalogue.json

```diff
@@ -997,28 +959,28 @@
         },
         {
           "title": "Leap",
-          "upgrade": "Soar",
-          "tier": "blue",
+          "upgrade": "Levitate",
+          "tier": "green",
           "details": {
-            "rules": "Jump 8 metres in the air",
+            "rules": "Jump 25 metres in the air",
             "flavour": "\"Good lord, look at him go..\" - Warner Higgins, concerned father",
             "image": "assets/core/card/illustration/Leap.png"
           }
         },
         {
-          "title": "Soar",
+          "title": "Levitate",
           "upgrade": "Ascend",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Jump 25 metres in the air",
-            "flavour": "\"I don't think I can even catch him at this point. He's on his own. Hear that? You're on your own buddy.\" - Warner Higgins, concerned father",
+            "rules": "Hover up to 5 feet off the ground until you lose concentration.",
```

Comment:
Trying to create differentiation between red and gold. This effect could be fun since you can make it last if you dodge everyting